doc/api/sqlite.md

```diff
@@ -241,6 +241,19 @@ Enables or disables the `loadExtension` SQL function, and the `loadExtension()`
 method. When `allowExtension` is `false` when constructing, you cannot enable
 loading extensions for security reasons.
 
+### `database.location([dbName])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `dbName` {string} Name of the database. This can be `'main'` (the default primary database) or any other
+  database that have been added with [`ATTACH DATABASE`][] **Default:** `'main'`.
+* Returns: {string} The location of the database file. When using an `in-memory` database,
+  this method returns an empty string.
```

Comment:
Can we return null for this case rather than empty string?

Reply:
+1 .. I think `null` would absolutely be better here.

Reply:
+1 .. I think `null` would absolutely be better here.

Reply:
will do, folks